crates/graphql/src/schema.rs

```diff
@@ -181,6 +181,50 @@ impl<'a> From<models::MetadataCreator<'a>> for NftCreator {
     }
 }
 
+#[derive(Debug, Clone)]
+struct NftAttribute {
+    metadata_address: String,
+    value: String,
+    trait_type: String,
+}
+
+impl NftAttribute {
+    pub fn metadata_address(&self) -> String {
+        self.metadata_address.clone()
+    }
+
+    pub fn value(&self) -> String {
+        self.value.clone()
+    }
+
+    pub fn verified(&self) -> String {
+        self.trait_type.clone()
+    }
+
+    pub async fn attribute(&self, ctx: &AppContext) -> Vec<NftAttribute> {
+        ctx.nft_attribute_loader
+            .load(self.metadata_address.clone())
+            .await
+    }
+}
+
+impl<'a> From<models::MetadataAttribute<'a>> for NftAttribute {
+    fn from(
+        models::MetadataAttribute {
+            metadata_address,
+            value,
+            trait_type,
+            ..
```

Comment:
I tried to bring in `id` but was unable to get the type `uuid:Uuid` to `use` correctly

Reply:
We should probably re-export it from the indexer_core crate to make that easier.  It’s randomly generated by postgres so it’s not a huge deal either way if we don’t get it.